src/ui/control/fullscreen_icon.svg

```diff
@@ -0,0 +1,18 @@
+<svg viewBox="0 0 29 29" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <defs>
+        <path id="expand" d="M7.5 5C5.75 5 5 5.75 5 7.5V13h1l1.5-3 5.5 4 1-1-4-5.5L13 6V5H7.5z"/>
```

Comment:
If we inline SVG, we should care about unique `id`(as they are just plain nodes) attributes: https://stackoverflow.com/a/37000951/643514 (https://css-tricks.com/youre-inlining-svg-icons-deal-unique-titles-ids/)

I suggest to prefix them with `mapbox-` at least. (in case it is not already done by `svgo` or something).